### DISCUSSION TITLE
How to set the DPI attribute of the generated PDF

### DISCUSSION THREAD

**Discussion by shidaiman** [cefsharp/CefSharp#4585] [Q&A]
I want to render HTML into a PDF file through command  'browser.PrintToPdfAsync', but the file always shows dpi=72, so I need to set it to 300 for printing. I can only manually modify this value through other software or during printing.This issue has been bothering me for 5 years. If cefsharp does not support it yet, I hope to be able to modify the DPI attribute.Looking forward to good results very much.

**Comment by amaitland** [comment]
Does Chrome support this? We can only provide features/options that Chrome supports.

From memory the underlying method that's called for `PrintToPdf` is the same as https://chromedevtools.github.io/devtools-protocol/tot/Page/#method-printToPDF
